.github/markdown-link-check-config.json

```diff
@@ -0,0 +1,15 @@
+{
+    "ignorePatterns": [
+        {
+            "pattern": "https://crypto.stackexchange.com/questions/54852/what-happens-if-a-sha-256-input-is-too-long-longer-than-512-bits"
```

Comment:
b/c this link is flakey

Reply:
yeah the link checker always thinks this link is dead, but we attempt to access it locally it works fine. this could be something on stackexchange's backend, or some other weird arbitrary reason. Its been causing all of our checks to die, so we either have to add something like this or get rid of problematic links all together